docs/api/post-v1-determineversion.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: Experimental determineversion
+title: POST /v1experimental/determineversion
```

Comment:
I think we specifically changed this to Experimental determineversion because of how badly this renders on the sidebar

Reply:
OK, I found a hack. Adding a space after the second slash causes the `determineversion` to be rendered on the next line. I think it's important to be consistent here.

Reply:
This is how it looks: ![image](https://github.com/google/osv.dev/assets/759062/e4a3475e-b66b-4b72-81b7-e8c447e341fe)

Reply:
Ah, looks much better!

Reply:
the title (the text on the browser tab) might look weird maybe, but most likely it'll be too long to show up anyway.

Reply:
It gets rendered with a newline very similarly to the side panel here for me.